tests/unit/Alerts.spec.ts

```diff
@@ -49,6 +51,10 @@ describe('Dashboard - UI', () => {
           get (): boolean { return this.$data._isFrozenInformationAlert },
           set (val: boolean) { this.$data._isFrozenInformationAlert = val }
         },
+        isInDissolutionAlert: {
```

Comment:
@severinbeauvais Just wanted to say, the test file here is absolutely gorgeous. It was so easy to add a new test case for the new alert 😃

Reply:
Thanks. I just noticed that this test suite can be optimized a bit by doing the setup and teardown just once ("all") instead of for each test case.

Before:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/11a959cc-cea8-43ef-9b71-333f50771d6c)

After:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/0e828004-2762-499f-97ff-9e6c90d8aca3)

Not a huge savings but seems like better code since it's not wasting cycles. Up to you if you want to update this.

Reply:
Done. Thanks for this!